.typos.toml

```diff
@@ -0,0 +1,21 @@
+[files]
+extend-exclude = [
+    ".git/",
+    "BACKERS.md",
+    "e2e/integration/drupal-baseline.neon",
+    "e2e/integration/phpunit-baseline.neon",
+    "website/src/errorsIdentifiers.json",
+    "website/src/js/data/countries.json",
```

Comment:
Please just include the whole website, including error identifiers. I doňt care about spelling errors anywhere else.

Oh, maybe also the README, thanks.

And also please add the same workflow in phpstan-src, to check src/ and README. Thanks.

Reply:
All done 🍏